files/build_templates/bgp.service.j2

```diff
@@ -3,12 +3,16 @@ Description=BGP container
 Requires=updategraph.service
```

Comment:
This file is causing conflict because it was recently moved to files/build_templates/single_instance/bgp.service.j2

Reply:
Fixed.